adventure/api.py

```diff
@@ -8,9 +8,12 @@
 from rest_framework.decorators import api_view
 import json
 
+#API calls used by the player as they navigate the dungeon and interact with other players
```

Comment:
Good comments like these will help you remain organized as your project grows. Well done!